File: 1/main.cpp
#include <iostream>
#include <cmath>

using namespace std;

bool isPrime(int x);

int main(void) {
    int sum = 0;
    for (int i = 2; i < 3000; i++)
        if (isPrime(i))
            sum += i;
    cout << sum << endl;
    return 0;
}

bool isPrime(int x) {
    for (int i = 2; i <= (int) sqrt(x); i++)
        if (x % i == 0)
            return false;
    return true;
}

File: 1/main.py
from math import sqrt
def isPrime(x):
    for i in range(2, int(sqrt(x) + 1)):
        if x % i == 0:
            return False
    return True;
sum = 0
for i in range(2, 3000):
    if isPrime(i):
        sum += i
print(sum)
